x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -587,3 +588,59 @@ export const relatedEventTotalForProcess: (
     };
   }
 );
+
+/**
+ * Total count of events related to `node`.
+ * Based on `ResolverNodeStats`
+ */
+export const totalRelatedEventCountForNode: (
+  state: DataState
+) => (nodeID: string) => number | undefined = createSelector(
+  relatedEventsStats,
+  (stats) => (nodeID: string) => {
+    const nodeStats = stats(nodeID);
+    return nodeStats === undefined ? undefined : nodeStats.events.total;
+  }
+);
+
+/**
+ * Count of events with `category` related to `nodeID`.
+ * Based on `ResolverNodeStats`
+ */
+export const relatedEventCountOfTypeForNode: (
+  state: DataState
+) => (nodeID: string, category: string) => number | undefined = createSelector(
+  relatedEventsStats,
+  (stats) => (nodeID: string, category: string) => {
```

Comment:
❔ So we're going back to thunks now?

Reply:
We haven't gotten rid of any thunks as far as I know. I think we should move to parametric selectors but I didn't take that step in this PR.